web/content/docs/benchmarks/EhlersDamage/EhlersDamage.md

```diff
@@ -0,0 +1,12 @@
++++
+date = "2017-02-16"
+author = "Francesco Parisio"
+title = "Ehlers plasti-damage coupled model"
+
+aliases = [ "/docs/benchmarks/" ] # First benchmark page
```

Comment:
Please remove that line at it just defines the first of all benchmark pages (which is already set).

I am preparing a pull request adding correct submenus, [see this](https://github.com/bilke/ogs/blob/84d578e10f3b676b14878550c5ca00f69e917e7f/web/README.md#setup-navigation-for-a-page) for usage.